src/Entity/Category.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CategoryRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Uid\Uuid;
+
+#[ORM\Entity(repositoryClass: CategoryRepository::class)]
+class Category
+{
+    #[
+        ORM\Id,
+        ORM\Column(type: 'uuid', unique: true, nullable: false)
+    ]
+    private Uuid $id;
+
+    #[ORM\Column(type: 'string', length: 255)]
```

Comment:
```suggestion
    #[ORM\Column]
```

They are the default values.